doc/intro.md

```diff
@@ -291,66 +286,66 @@ concept: _cut-through_. With this addition, a Mimblewimble chain gains:
 * Extremely good scalability, as the great majority of transaction data can be
   eliminated over time, without compromising security.
 * Further anonymity by mixing and removing transaction data.
-* And the ability for new nodes to sync up with the rest of the network very
-  efficiently.
```

Comment:
Why remove it? It's a bit of a repetition of point 1 but can still clarify things here no?

Reply:
I removed it beacuse it was repetition of the first point, yes. If you want to keep it I would suggest merging it together with the first point rather than having it separate as it strikes me as dishonest.

For example: *"Extremely good scalability as the great majority of transaction data can be eliminated over time without compromising security, decreasing the burden of running a full node as well as streamlining the synchronization of new nodes."*